### PR TITLE
GITHUB=APD-187: implement serverless function to get list of updates and is there is new updates

### PR DESCRIPTION
**Pull request by ahocquard**

### CODE REVIEW

File: src/firebase/functions/index.js [hunk 1]
```diff
@@ -1,22 +1,106 @@
...
+        .get()
         .then(snapshot => {
-            const announcements = [];
+            let hasNew = false;
 
-            snapshot.forEach(doc => {
-                announcements.push(doc.data());
+            // it is not possible to filter with two range operators in firebase, so it simulates it
+            snapshot.forEach(announcement => {
+                if (announcement.data().startDate <= formattedCurrentDate  ) {
```
Comment (christophe-chausseray) [inline src/firebase/functions/index.js:95]:
```suggestion
                if (announcement.data().startDate <= formattedCurrentDate) {
```

File: tasks/build-monthly-updates-as-json.js [hunk 2]
```diff
@@ -149,3 +152,38 @@ function getBase64Content(file, imageRelativePath) {
...
+ */
+function transformToPimEditions(editions) {
+    return editions.map(edition => {
+        return edition === 'EE' ? 'Serenity' : edition;
+    });
+}
+
+function getNotificationEndDate(starDateString) {
+    let startDate = new Date(starDateString);
+    startDate.setDate(startDate.getDate() + 7);
```
Comment (christophe-chausseray) [inline tasks/build-monthly-updates-as-json.js:169]: I think + 7 should not be hardcoded. But we should have a dynamic notification period instead, no ?

Comment (ahocquard) [reply tasks/build-monthly-updates-as-json.js:169]: The rule is +7 for the updates. So no need to have dynamic period. And I don't like ocnstant when it's not used anywhere else (it's simple like that)

File: src/firebase/functions/index.js (hunk 1 above)
Comment (ahocquard) [reply src/firebase/functions/index.js:95]: I will activate the linter when executing tests in another PR

**Pull request merged**